src/actions/breakpoints.js

```diff
@@ -239,3 +244,52 @@ export function setBreakpointCondition(
     });
   };
 }
+
+export function toggleBreakpoint(line: number, column?: number) {
+  return ({ dispatch, getState, client, sourceMaps }: ThunkArgs) => {
+    const selectedSource = getSelectedSource(getState());
+    const bp = getBreakpointAtLocation(getState(), { line, column });
+
+    if (bp && bp.loading) {
+      return;
+    }
+
+    if (bp) {
+      // NOTE: it's possible the breakpoint has slid to a column
```

Comment:
do we want to slide a breakpoint to a column if the user is expecting a line breakpoint?